tests/common.rs

```diff
@@ -20,6 +20,12 @@ pub fn initialize_logging() {
     let _ = INIT_LOGGING;
 }
 
+pub fn get_blockfrost_client() -> BlockfrostAPI {
+    let settings = BlockFrostSettings::default();
+
+    BlockfrostAPI::new("previewTjUg7ty9Har2JdaRYlzsGs7Wsy6wp8G6", settings)
```

Comment:
How sensitive is this key? Maybe a new one should be an env var, and a GitHub secret, and this one revoked, as this is a public repo.

Reply:
This account was generated specifically for these tests, and you can create a new free account in less than 30 seconds. I don't want to add more environments just for testing. This applies to both keys.

Reply:
Fair enough!